addons/hr/views/hr_department_views.xml

```diff
@@ -43,12 +44,15 @@
             <field name="name">hr.department.tree</field>
             <field name="model">hr.department</field>
             <field name="arch" type="xml">
-                <tree string="Companies" sample="1">
+                <tree string="Companies" sample="1" editable="bottom">
+                    <field name="company_id" column_invisible="True"/>
```

Comment:
Why is this company_id field here when there is another one further down?

Reply:
Yaa, we need this because company field is used in manager field's domain so when we are in single company env then it'll give traceback company field is missing
That's why we need this :slightly_smiling_face: